go.sh

```diff
@@ -26,29 +26,35 @@ if [ "${LOCATIONS_CSV}" == "" ]; then
     LOCATIONS_CSV=/
 fi
 
+NAXSI_LISTEN_PORT="10418"
```

Comment:
Note: If we make this configurable then the README will need updating.

Also, we could merge for now and make these improvements later.